digiroad2-viite/src/test/scala/fi/liikennevirasto/viite/dao/RoadwayDAOSpec.scala

```diff
@@ -1100,5 +1100,39 @@ class RoadwayDAOSpec extends FunSuite with Matchers {
 
     }
   }
+  
+  test("Test When fetching road part history for road address browser then return history information for the road part") {
+    runWithRollback {
+      val roadNumber = 76
+      val roadPartNumber = 1
+      val roadwayNumber = Sequences.nextRoadwayNumber
+      val roadwayNumber2 = Sequences.nextRoadwayNumber
+      val rwHistoryRowStartDate = "1992-10-08"
+      val rwHistoryRowEndDate = "2022-11-14"
+      val rwCurrentRowStartDate = "2022-11-15"
+      val afterChangesSituationDate = "2022-11-15"
+      val historyChangesSituationDate = "2022-11-14"
+
+      val roadways = Seq(
```

Comment:
Ymmärtämistä nopeuttais vielä, jos selittäis roadwayt lyhyesti auki.
[Irtoaa nyt vaan enempivähempi kökköjä lauseita, mutta] esim.:
// a road part gets elongated at rwCurrentRowStartDate